packages/did/src/Web3Names/web3names.chain.ts

```diff
@@ -0,0 +1,104 @@
+/**
```

Comment:
M: The file name must start with capital W.

Reply:
fixed